src/utils.ts

```diff
@@ -57,3 +58,40 @@ export function callEachWith (callbacks: Function[], arg0?: any) {
     cb(arg0)
   }
 }
+
+const isBrowser = typeof navigator !== 'undefined'
+
+/** Start debugging hook names and timing in console */
+export function createDebugger (hooks: Hookable, { tag, inspect = isBrowser, group = isBrowser, filter: _filter = () => true }: CreateDebuggerOptions = {}) {
```

Comment:
Style: Can we please avoid destructure options?

Reply:
as you wish

Reply:
out of curiosity, why?

Reply:
I destructured them for two reasons, FYI. First, to avoid the need for an intermediate variable assignment for `_filter` and, second, to prevent them from being mutable - e.g. a user _changing_ the options tag down the line.

Reply:
Mainly consistency. I use this pattern in general which also helps to consistently pass options or context with multiple functions without partially picking.

For applying defaults, we can use an option assignment...